### DISCUSSION TITLE
Features? What features do you like so see?

### DISCUSSION THREAD

**Discussion by optimismus** [cbrosius/luxtronik_v1#22] [Ideas]
Hi, maybe it is helpful for Christian to see, which features are requested more than others from the community.

These are mine for now:

1. Set Heizung- and Warmwassermode (Auto, Aus, Ferien ...)
2. Set SOLL temperatures (Brauchwasser SOLL, Rücklauf SOLL)
3. more security (encryption key, ota key ... by default)
4. get error messages from heatpump (seems to be not that easy --> discussion for IO Broker module)
5. get all values from heatpump (e.g. # of hours, # of cycles ... for calculating if it running good (longer cycles are better))
6. full ipv6 support

**Comment by approx3145** [comment]
Das Lesen der Betriebsstunden wäre cool
Das Setzen der Rücklauf Soll Temperaturwäre super

**Comment by cbrosius** [comment]
Habe jetzt mal das Auslesen der Betriebsstunden hinzugefügt.

Beim Schreiben von Werten bin ich mir immer noch nicht sicher, ob das so eine gute Idee ist.

**Reply by cbrosius** [reply]
Hier mal ein Link zur Umfrage, ob ich das Schreiben einbauen soll: https://github.com/cbrosius/luxtronik_v1/discussions/30
